ssl/custom-ca.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Should we make this a GitHub secret? I know it's just for testing but I'm afraid scanners might pick this up and open issues in the repo.